packages/itwinui-css/backstop/tests/button.html

```diff
@@ -1072,338 +1072,278 @@ <h2>Grouped</h2>
+          ></svg-delete>
+        </button>
+        <button
+          disabled
+          title="Rename"
+          class="iui-button"
+        >
+          <svg-rename
+            class="iui-button-icon"
+            aria-hidden="true"
+          ></svg-rename>
+        </button>
+        <button
+          disabled
+          title="More options&mldr;"
+          class="iui-button"
+        >
+          <svg-more
+            class="iui-button-icon"
+            aria-hidden="true"
+          ></svg-more>
+        </button>
       </span>
 
       <br />
 
       <span class="iui-button-group">
-        <div>
-          <button
-            title="List view"
-            class="iui-button"
-          >
-            <svg-list
-              class="iui-button-icon"
-              aria-hidden="true"
-            ></svg-list>
-          </button>
-        </div>
-        <div>
-          <button
-            title="Tile view"
-            class="iui-button"
-            data-iui-active="true"
-          >
-            <svg-thumbnails
-              class="iui-button-icon"
-              aria-hidden="true"
-            ></svg-thumbnails>
-          </button>
-        </div>
+        <button
+          title="List view"
+          class="iui-button"
+        >
+          <svg-list
+            class="iui-button-icon"
+            aria-hidden="true"
+          ></svg-list>
+        </button>
+        <button
+          title="Tile view"
+          class="iui-button"
+          data-iui-active="true"
+        >
+          <svg-thumbnails
+            class="iui-button-icon"
+            aria-hidden="true"
+          ></svg-thumbnails>
+        </button>
       </span>
 
       <br />
 
       <span class="iui-button-group">
-        <div>
-          <button
-            disabled
-            title="List view"
-            class="iui-button"
-          >
-            <svg-list
-              class="iui-button-icon"
-              aria-hidden="true"
-            ></svg-list>
-          </button>
-        </div>
-        <div>
-          <button
-            disabled
-            title="Tile view"
-            class="iui-button"
-            data-iui-active="true"
-          >
-            <svg-thumbnails
-              class="iui-button-icon"
-              aria-hidden="true"
-            ></svg-thumbnails>
-          </button>
-        </div>
+        <button
+          disabled
+          title="List view"
+          class="iui-button"
+        >
+          <svg-list
+            class="iui-button-icon"
+            aria-hidden="true"
+          ></svg-list>
+        </button>
+        <button
+          disabled
+          title="Tile view"
+          class="iui-button"
+          data-iui-active="true"
+        >
+          <svg-thumbnails
+            class="iui-button-icon"
+            aria-hidden="true"
+          ></svg-thumbnails>
+        </button>
       </span>
 
       <br />
 
       <span class="iui-button-group-vertical">
```

Comment:
Can this be changed to `class="iui-button-group" data-iui-orientation="vertical"`?

Reply:
yeah, totally

Reply:
i was working on this but it feels a bit repetitive and verbose because:
- the attribute is applied on the button-group but most of the styles are applied to nested children
- the border-radius properties are different for vertical and horizontal